subgraphs/_reference_/src/common/constants.ts

```diff
@@ -23,6 +26,27 @@ export namespace Network {
   export const XDAI = "XDAI";
 }
 
+// The network names corresponding to the ones in `dataSource.network()`
+// They should mainly be used for the ease of comparison.
+// See below for a complete list:
+// https://thegraph.com/docs/en/hosted-service/what-is-hosted-service/#supported-networks-on-the-hosted-service
+export namespace SubgraphNetwork {
```

Comment:
Missing "harmony" and "cronos"

Reply:
yeah, I don't think these two are supported by The Graph yet

Reply:
see https://thegraph.com/docs/en/hosted-service/what-is-hosted-service/#supported-networks-on-the-hosted-service

Reply:
I don't mind removing them from the list above